### PR TITLE
Updated as of Bans, Circa 14-03-19

### PR DESCRIPTION
**Pull request by Monohell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
 
 ## Banlist
 
-_Correct as of 2019-03-13_ 
+_Correct as of 2019-03-14_ 
 
 - [Standard Commander ban list](https://scryfall.com/search?q=banned%3Aedh&order=name&as=grid&unique=cards)
+- Commanders with Keyword: "Partner" (You may play one of the partner commanders, but you may not have both in the command zone)
```
Comment (cmeister2) [inline README.md]: Isn't this covered by line 9?

File: README.md [hunk 2]
```diff
@@ -10,15 +10,17 @@ You are only permitted to have *one* Commander. Commanders with Partner may be u
...
 
 plus
 
 - [Cranial Plating](https://scryfall.com/card/mm2/206/cranial-plating)
 - [Demonic Tutor](https://scryfall.com/card/lea/104/demonic-tutor)
 - [Empty the Warrens](https://scryfall.com/card/dds/15/empty-the-warrens)
 - [Frantic Search](https://scryfall.com/card/uma/57/frantic-search)
+- [Library of Alexandria](https://scryfall.com/card/vma/303/library-of-alexandria)
```
Comment (cmeister2) [inline README.md]: Library of Alexandria is already in the standard Commander ban list and as such does not need explicitly calling out.

Comment (Monohell) [reply README.md]: Fair shout, Didn't see it on my prior perusal.

File: README.md (hunk 1 above)
Comment (Monohell) [reply README.md]: I'd use this in favour of Line 9, or change Line 9 to reflect this. One of the instances is redundant, but additional clarity can't hurt.

Comment (cmeister2) [reply README.md]: In that case I'd change this line to say `Multiple Commanders with Keyword: "Partner"` to better clarify that you can still run Commanders with Partner, just not two of them.

Comment (Monohell) [reply README.md]: - Multiple Commanders with the Keyword "Partner" You may still play cards with the "Partner" Keyword, however only one card with the "Partner" Keyword may be in your command zone

Proposed Change above, or is this just worse?

Comment (cmeister2) [reply README.md]: Seems better, yeah!

**Pull request merged**